python/docs/source/conf.py

```diff
@@ -14,6 +14,8 @@
 #
 import os
 import sys
+import vowpalwabbit
```

Comment:
This assumes the installed version of vowpalwabbit is the one you're generating documentation right? I think that's okay though

Reply:
I have changed this by keeping in view that the user wouldn't use the doc file. It is at the development end which is synced with the current master and is always updates with the new versions.

Reply:
This will also help the users to get updated with the newer versions. As it will allow it to compare their current version in their local computers to the version of the docs provided online